gradle/libs.versions.toml

```diff
@@ -0,0 +1,89 @@
+[versions]
+kotlin = "1.7.10"
+kotlin-coroutines = "1.6.4"
+java-target = "1.8"
+
+# Note: there are some compose/androidx versions left hardcoded in `paparazzi/paparazzi/build.gradle`,
+# because they refer to JAR files names in the `paparazzi/paparazzi/libs` folder.
+androidx-compose-compiler = "1.3.0"
+androidx-compose-ui-material = "1.2.1"
+androidx-lifecycle = "2.4.0"
+androidx-annotations = "1.3.0"
+
+agp = "7.1.2"
+mavenPublish = "0.21.0"
+gradle-versions = "0.42.0"
+grgit = "5.0.0"
+spotless = "6.8.0"
+ktlint = "0.46.1"
+
+tools-common = "27.1.2"
+tools-sdkCommon = "26.6.4"
+tools-layoutlib = "27.2.2"
+tools-ninepatch = "30.1.2"
+# Maps to this commit: https://android.googlesource.com/platform/prebuilts/studio/layoutlib/+/fa3aa65
+# which runs on JDK 11.
+layoutLibPrebuiltSha = 'fa3aa65'
+layoutlib = "2021.2.1-patch1-fa3aa65"
+
+
+guava = "31.0.1-jre"
+kxml2 = "2.3.0"
+okio = "3.1.0"
+moshi = "1.13.0"
+jcodec = "0.2.5"
+bytebuddy = "1.12.10"
+
+junit = "4.13.2"
+testParameterInjector = "1.8"
+truth = "1.1.3"
+assertj = "3.23.1"
+
+[libraries]
+layoutlib-native-jdk11 = { module = "app.cash.paparazzi:layoutlib-native-jdk11", version.ref = "layoutlib" }
+layoutlib-native-macArm = { module = "app.cash.paparazzi:layoutlib-native-macarm", version.ref = "layoutlib" }
+layoutlib-native-macOsX = { module = "app.cash.paparazzi:layoutlib-native-macosx", version.ref = "layoutlib" }
+layoutlib-native-windows = { module = "app.cash.paparazzi:layoutlib-native-win", version.ref = "layoutlib" }
+layoutlib-native-linux = { module = "app.cash.paparazzi:layoutlib-native-linux", version.ref = "layoutlib" }
+
+tools-common = { module = "com.android.tools:common", version.ref = "tools-common" }
+tools-layoutlib = { module = "com.android.tools.layoutlib:layoutlib-api", version.ref = "tools-layoutlib" }
+tools-ninepatch = { module = "com.android.tools:ninepatch", version.ref = "tools-ninepatch" }
+tools-sdkCommon = { module = "com.android.tools:sdk-common", version.ref = "tools-sdkCommon" }
```

Comment:
sort all libraries, but keep groups by prefix

Reply:
is this what you meant? d74b6ad2

Reply:
yep!